src/xrCore/_std_extensions.cpp

```diff
@@ -15,7 +15,12 @@ char* timestamp(string64& dest)
 {
     time_t     now = time(nullptr);
     struct tm  tstruct;
-    tstruct = *localtime(&now); // localtime is NOT thread-safe
+
+#if defined(WINDOWS)
+    localtime_s(&tstruct, &now);// thread-safe for windows
+#else
+    localtime_r(&now, &tstruct);// thread-safe for posix systems
```

Comment:
We can also use localtime_s for POSIX. But it's not thread-safe there, right?

Reply:
No, POSIX doesn't provide the localtime_s funtion.
localtime_s is a windows implementation for localtime_r.

https://pubs.opengroup.org/onlinepubs/009695399/functions/localtime.html

Reply:
Ah, ok.